examples/asr/speech_to_text_rnnt_bpe.py

```diff
@@ -75,6 +77,13 @@ def main(cfg):
     exp_manager(trainer, cfg.get("exp_manager", None))
     asr_model = EncDecRNNTBPEModel(cfg=cfg.model, trainer=trainer)
 
+    # initialize the model with the weights of the checkpoint specified by 'load_weights_from_checkpoint' in the configs
+    # You may use this option to start the training from a pre-trained checkpoint
+    checkpoint_path = cfg.model.get("load_weights_from_checkpoint", None)
+    if checkpoint_path:
+        logging.info(f'Initializing the model with the checkpoint at "{checkpoint_path}"')
+        asr_model.load_state_dict(torch.load(checkpoint_path, map_location=asr_model.device)["state_dict"])
+
```

Comment:
Load checkpoint on seperate line, load weights and then explicitly del the entire checkpoint. Otherwise the optimizer state will stay over and won't get deleted before model creation + training begins.